### PR TITLE
Resolved Bug: Sroll Bar starts in the middle

### PR DESCRIPTION
**Pull request by komalioruganti**
## Related Issue

 [Bug]: Project Info page opens with scroll bar in the middle  #755

## Description

The bug (Scroll bar starts form the middle of the page when "More" is clicked on Project cards) has been resolved.

## Screenshots

https://github.com/priyankarpal/ProjectsHut/assets/95839946/9b3b6759-9b5c-4c79-89b4-a1d0e7b328df

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -1901,5 +1901,24 @@
...
+      "Projects": [
+        {
+          "link": "https://github.com/komalioruganti/ChatGPT-Clone",
+          "title": "ChatGPT-Clone",
+          "description": "A bot that generates text and images for the given prompts.",
+          "tech": ["React","Node","Express","JavaScript","CSS","OpenAi API"]
+        }
+      ]
+    }
+
```
Comment (priyankarpal) [inline src/DB/projects.json:1923]: you already have added this project so please remove the code

**Review by priyankarpal — CHANGES_REQUESTED** [review]
could you share your progress on this issue?

**Comment by komalioruganti** [comment]
I will make a new pull request otherwise

**Comment by komalioruganti** [comment]
I think I made a mistake. Can you close this pull request... I will make a new pull request. I am contributing for the first time. I am very sorry

**Comment by komalioruganti** [comment]
I have made a new pull request. Check this one 
Bug resolved: Scroll bar start from middle of page #1012

**Pull request closed without merging**